### PR TITLE
Added validate designs under XRT tests folder

### PR DESCRIPTION
**Pull request by vishnuchebrolu**

### CODE REVIEW

**Comment by AShivangi** [comment]
retest this please

**Comment by maxzhen** [comment]
ok to test

File: tests/validate/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+find_library(xrt_core_LIBRARY
+  NAMES ${XRT_CORE_LIBRARY}
+  HINTS "${XILINX_XRT}/lib")
+message("xrt_core_LIBRARY=${xrt_core_LIBRARY}")
+
+find_library(xrt_coreutil_LIBRARY
+  NAMES xrt_coreutil
+  HINTS "${XILINX_XRT}/lib")
+message("xrt_coreutil_LIBRARY=${xrt_coreutil_LIBRARY}")
```
Comment (stsoe) [inline tests/validate/CMakeLists.txt:25]: Where is linking with the core XRT library needed?  Can this be removed

File: tests/validate/bandwidth_test/src/host.cpp [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+#include <xcl2.hpp>
+#include <boost/property_tree/ptree.hpp>                                        
+#include <boost/property_tree/json_parser.hpp>
+
+#define TYPESIZE 512
+const size_t typesize = TYPESIZE;
+
+double getMicroTime() {
+  struct timeval currentTime;
+  gettimeofday(&currentTime, NULL);
```
Comment (stsoe) [inline tests/validate/bandwidth_test/src/host.cpp:46]: No NULL in C++, please use nullptr.

File: tests/validate/common.mk [hunk 3]
```diff
@@ -0,0 +1,116 @@
+#lowlevel suite common makefile definitions
+
+ifdef XILINX_SDACCEL
+ XILINX_VITIS := $(XILINX_SDACCEL)
+endif
```
Comment (stsoe) [inline tests/validate/common.mk:5]: Can we clean this up?  There is no SDACCEL, do we need to support folks setting this var?

**Comment by dayeh-xilinx** [comment]
retest this please

**Pull request merged**